source/_components/google_assistant.markdown

```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
 
 ### {% linkable_title Setup %}
 
-1. Create a new project in the [developer console](https://console.actions.google.com/).
-  a. Add/Import project
-  b. Click on `BUILD` on the `Smart home` card
```

Comment:
This should read:
Click on `Home Control` card, select `Smart home` recommendation